components/shortcuts_modal.jsx

```diff
@@ -131,6 +131,16 @@ const allShortcuts = defineMessages({
         id: t('shortcuts.msgs.reply'),
         defaultMessage: 'Reply to last message in channel:\tShift|Up',
     },
+    msgReaction: {
+        default: {
+            id: t('shortcuts.msgs.reaction'),
+            defaultMessage: 'React to last message: \tCtrl|Shift| \\\\', //eslint-disable-line
```

Comment:
I was able to reproduce the issue on:
- Linux/Ubuntu: Firefox and Chrome
- Windows 10: Firefox and Chrome

But interestingly I wasn't able to reproduce with my local dev environment.

I read through the code and might found the debatable code line. Applying this change to my dev environment shows a correct result. Do you think this could be the cause of the problem? If so, I propose to apply the change and set up a new test server.

```suggestion
            defaultMessage: 'React to last message: \tCtrl|Shift| \\', //eslint-disable-line
```

Reply:
@hanzei  surprisingly it is working fine in Mac too now, it tried this previously with only ``\\`` slashes but at that time Mac was creating issues, but strangely now it is working fine. So making changes to the code and updating the PR.